### PR TITLE
Refactoring - moving ingest field name encoding into one function

### PR DESCRIPTION
**Pull request by pdelewski**
This PR refactor ingest workflow and unifies ingest field encoding inside `processInsertQuery` function.
That's not finish, it's just a step forward

### CODE REVIEW

File: quesma/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -518,42 +518,79 @@ func (lm *LogManager) generateNewColumns(
 	return alterCmd
 }
 
-func generateNonSchemaFieldsString(attrsMap map[string][]interface{}) (string, error) {
+// This struct contains the information about the columns that are part of the schema
```
Comment (mieciu) [inline quesma/clickhouse/clickhouse.go:521]:
```suggestion
// This struct contains the information about the columns that aren't part of the schema
```
😉

File: quesma/clickhouse/clickhouse.go [hunk 2]
```diff
@@ -518,42 +518,79 @@ func (lm *LogManager) generateNewColumns(
...
+				value = nonSchemaFields[i].Type
+			} else {
+				value = nonSchemaFields[i].Value
+			}
+			if i > 0 {
+				nonSchemaStr += ","
+			}
+			nonSchemaStr += fmt.Sprintf("\"%s\":\"%s\"", nonSchemaFields[i].Key, value)
+		}
+		nonSchemaStr = nonSchemaStr + "}"
```
Comment (mieciu) [inline quesma/clickhouse/clickhouse.go:552]: Hmm would StringBuilder make this look a little better? I'm nor sure myself, I'll leave that up to you :)

**Review by mieciu — APPROVED** [review]
One note on StringBuilder, but if you don't feel like doing it within this PR, I'm fine with the code as is 🙂

File: quesma/clickhouse/clickhouse.go (hunk 2 above)
Comment (pdelewski) [reply quesma/clickhouse/clickhouse.go:552]: Yes, good point

**Pull request merged**